packages/manager/apps/pci-quota/src/pages/quota/IncreaseQuotaSupport.page.tsx

```diff
@@ -0,0 +1,266 @@
+import { z } from 'zod';
+
+import { Button, Textarea } from '@datatr-ux/uxlib';
+import { zodResolver } from '@hookform/resolvers/zod';
+
+import { usePciUrl } from '@ovh-ux/manager-pci-common';
+import {
+  Links,
+  LinkType,
+  Notifications,
+  Title,
+  useMe,
+  useNotifications,
+} from '@ovh-ux/manager-react-components';
+import { useTracking } from '@ovh-ux/manager-react-shell-client';
+import { OdsMessage } from '@ovhcloud/ods-components/react';
+import { Controller, useForm } from 'react-hook-form';
+import { Translation, useTranslation } from 'react-i18next';
+import { useHref, useNavigate, useParams } from 'react-router-dom';
+
+import { useMemo } from 'react';
+import { TCreateTicketResponse } from '@/api/data/ticket';
+import {
+  useGetIssueTypes,
+  useMutationCreateTicket,
+} from '@/api/hooks/useIssueTypes';
+import {
+  SUPPORT_ISSUE_TYPE_IDS,
+  SUPPORT_TICKET_ID_URL,
+  TRACK,
+} from '@/constants';
+
+const MIN_CHARACTERS = 20;
+const IncreaseQuotaForm = ({
+  fields,
+  onSubmit,
+  isPending,
+}: {
+  fields: { id: number; label: string; isRequired: boolean }[];
+  goBackHref: string;
+  onSubmit: (data: { [id: number]: string }) => void;
+  isPending: boolean;
+}) => {
+  const { t } = useTranslation('quotas/increase');
+  const object = {};
+
+  fields?.forEach((field) => {
+    object[field.id] = field.isRequired
+      ? z.string().min(
+          MIN_CHARACTERS,
+          t('pci_projects_project_quota_increase_error_message_min', {
+            min: MIN_CHARACTERS,
+          }),
+        )
+      : z.string().optional();
+  });
+
+  const schema = z.object(object);
+  const defaultValues = useMemo(
+    () =>
+      fields.reduce((acc, field) => {
+        acc[field.id] = '';
+        return acc;
+      }, {} as Record<string, string>),
```

Comment:
`as Record<string, string>` could be avoided by typing the reduce function accumulator: 
```ts
fields.reduce<Record<string, string>>((acc, field) => {
```